test/zero-config/entry-present/zero-config.test.js

```diff
@@ -0,0 +1,12 @@
+const { run } = require('../../utils/test-utils');
+
+describe('Zero Config tests', () => {
+    it('runs when no config is supplied but entry is present', () => {
```

Comment:
We could also test for existence of bundle output file.

Reply:
We can def do that, thinking if this would be the right scope for it as output file check is something which is config/flag agnostic. wdyt? Since we're not checking for it anywhere else, should be also checked for it in all other tests?

Reply:
Or maybe we can just check the test util since it's used everywhere, will add tests for the `run` fn that we use in all tests separately. Would make more sense to check file existence there.

Reply:
In this particular case we should test it either way imo.  Because the test is for checking whether without config we are bundling or not.

Tests for run as well in a different PR would be great :)

Reply:
> Or maybe we can just check the test util since it's used everywhere, will add tests for the `run` fn that we use in all tests separately. Would make more sense to check file existence there.

We're consuming the `run` fn for couple of test cases where bundle output is not generated, say the `migrate` command. It should be taken into account while writing the respective tests.

Reply:
Sounds fair, added. 👍